ingestion-beam/src/test/java/com/mozilla/telemetry/transforms/WithCurrentTimestampTest.java

```diff
@@ -0,0 +1,21 @@
+package com.mozilla.telemetry.transforms;
+
+import static org.junit.Assert.assertEquals;
+
+import org.joda.time.Instant;
+import org.junit.Test;
+
+public class WithCurrentTimestampTest {
+
+  @Test
+  public void testAdjustedTimestamp() {
+    Instant startTimestamp = new Instant();
+    assertEquals(1.0 * startTimestamp.getMillis(),
+        1.0 * WithCurrentTimestamp.adjustedTimestamp(new Instant(0)).getMillis(), 100.0);
```

Comment:
Is the 100 for the delta?  Shouldn't we expect no delta since the instant should be the same?

Reply:
I should have added a comment here. I would expect the transform to create its own `new Instant()` which will probably be a few millis after `startTimestamp`.